src/routes/UserRouter.ts

```diff
@@ -2,11 +2,23 @@
 
 import express from "express";
 import { getSingleJobRole, getAllJobRoles } from "../controllers/JobRoleController";
+import { getUploadForm, postUpload } from "../controllers/ApplicationController";
 import { allowRoles } from "../middleware/AuthMiddleware";
 import { UserRole } from "../models/JwtToken";
+import multer from "multer";
+import { multerConfig } from "../multerConfig";
+
+const upload = multer(multerConfig);
 
 export const userRouter = express.Router();
 
+userRouter.get('/job-roles', allowRoles([UserRole.Admin, UserRole.User]), getAllJobRoles);
 userRouter.get('/job-roles/:id', allowRoles([UserRole.Admin, UserRole.User]), getSingleJobRole);
 
-userRouter.get('/job-roles', allowRoles([UserRole.Admin, UserRole.User]), getAllJobRoles);
+userRouter.get('/apply/:id', allowRoles([UserRole.Admin, UserRole.User]),getUploadForm);
+userRouter.post('/apply/:id',allowRoles([UserRole.Admin, UserRole.User]),upload.single('file'), postUpload);
+
+userRouter.get('/upload-success', allowRoles([UserRole.Admin, UserRole.User]),async (req: express.Request, res: express.Response): Promise<void> => {
+  res.render('apply-succesful.html');
```

Comment:
This should render a nunjucks file and not a html file as it doesn't exist

Reply:
This has been changed to a nunchucks file as requested

Reply:
This has now been removed as it wasn't needed either